File: hacaton-front/src/domain/TestTask.js
import { createGuid } from '@/utils/utils';

export default class TeacherTask {

    constructor(data = {}) {
        console.log(data);
        this.id = data.id || createGuid();
        this.input = data.input || null;
        this.output = data.output || null;
    }
}

File: hacc-back-cb87f7f3af84e47c1bff440981eb203cbe86da15/src/main/java/app/entity/TestTask.java
package app.entity;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(schema = "public", name = "test_task")
public class TestTask {

    @Id
    @Column(name = "id", nullable = false)
    private String id;

    @Column(name = "task_id", nullable = false)
    private String taskId;

    @Column(name = "output", nullable = false)
    private String output;

    @Column(name = "input", nullable = false)
    private String input;

}
